pages/users/[id].tsx

```diff
@@ -167,8 +187,12 @@ export default function User({ loginContext }: Props) {
   ).toLocaleString()
   const weeklyPoints = $weeklyMetrics.points.toLocaleString()
 
+  const tweetText = `Iron Fish Incentivized Testnet: ${
```

Comment:
Feedback definitely wanted on this copy.

Reply:
I'm not sure about the hashtags beyond `#ironfish`, but otherwise seems pretty simple/unopinionated so people can customize it at will, which I like